app/loginManager.py

```diff
@@ -15,14 +14,22 @@ def authenticate(username, password) -> bool:
     user = UsersRepository.get_by_username(username)
 
     if user:
+        logger.debug("Authenticating a user, username exists in database")
+        logger.debug("Authenticating a user, checking password")
         if user.password == hash_password(password):
-            logger.debug("User is authenticated and logged into session")
             session['user'] = user.to_json()
+            logger.debug("User is authenticated and logged into session")
```

Comment:
logged into session sounds weird, you log in to application and session is created